src/main/java/net/openhft/chronicle/queue/impl/single/SingleChronicleQueueBuilder.java

```diff
@@ -1192,7 +1192,7 @@ public SingleChronicleQueueBuilder setAllNullFields(@Nullable SingleChronicleQue
     }
 
     public WriteLock appendLock() {
-        return readOnly() ? WriteLock.NO_OP : new TableStoreWriteLock(metaStore, pauserSupplier(), timeoutMS() * 3 / 2, TableStoreWriteLock.APPEND_LOCK_KEY);
+        return readOnly() ? WriteLock.NO_OP : new AppendLock(metaStore, pauserSupplier(), timeoutMS() * 3 / 2, TableStoreWriteLock.APPEND_LOCK_KEY);
```

Comment:
Perhaps the constant should be taken from AppendLock as well?

Reply:
Great idea. Done